src/Arcus.Messaging.Abstractions/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -21,8 +20,6 @@ public static class IServiceCollectionExtensions
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="services"/> is <c>null</c>.</exception>
         public static MessageHandlerCollection AddMessageRouting(this IServiceCollection services)
         {
-            Guard.NotNull(services, nameof(services), "Requires a set of services to add the message routing");
-
             MessageHandlerCollection collection = AddMessageRouting(services, configureOptions: null);
```

Comment:
Just wondering why we remove the Guard.NotNull statement and do not replace it by any other check ?

Reply:
To limit references, only guard-check what you yourself need.